pages/verify/[discordServerId]/[discordMemberId]/[customLink].tsx

```diff
@@ -154,11 +156,15 @@ const VerifyPage = ({ discordServerName, starknetNetwork }: Props) => {
 
 export async function getServerSideProps({ res, query }: any) {
   await setupDb();
-  let discordServerName = null;
+  let DiscordServerName = null;
```

Comment:
why did you add a uppercase also? Please try to keep the same coding practices as the rest of the repo